docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +143,31 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'polygon'`:
+   * Required attributes:
+     * `points` - list of points defining vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygon that affected by 'follow'. Could be a single bools to affact allpoints too. Default means that every point is affacted.
+     * `mode` - how those points are connected. may be "polygon"(default),"strip" or "triangles". "polygon" means that it will be viewed as vertices of a polygon center on the first one. "strip" means that it will be viewed as a triangles strip. "triangles" means that it will be viewed as some triangles that are not related to each other (therefor length of `points` in this mode have to be a multiple of 3).
+     * `inner` - if `true` it will make the inner edges be drawn as well. 
+     * `doublesided` - if `true` it will make the shapes visible from the back as well. Default is `true`. 
+
+ * `'block'` \ `'item'`:
+   * Required attributes:
+     * `pos` - position of the object.
+     * `item` - the object to show. could have NBT. is a item tuple or a string-id (exclusive to `'item'`)
+     * `block` - the object to show. could have NBT. is a block value or a string-id (exclusive to `'block'`)
+   * Optional attributes:
+     * `facing` - object's direction, where its facing. Possible options are: `player` (default, it
```

Comment:
the default should be 'north' for blocks, and 'player' facing makes the blocks rotate when looking at blocks off axis. It does not rotate in 3rd person mode indicating something its flipped with the angles with 'player' facing setting.

Reply:
with items, we could leave default 'north' for consistency.